content/blog/crossing-realities-ready-to-daydream.md

```diff
@@ -0,0 +1,117 @@
+
+### 5. Get and import the GoogleVRForUnity.unitypackage
+To get all the Daydream components for Unity you need to import the *GoogleVRForUnity.unitypackage*. You can get it [here](https://github.com/googlevr/gvr-unity-sdk/blob/master/GoogleVRForUnity.unitypackage "GoogleVRForUnity.unitypackage"), just click on `Download` on the right.
+When it's downloaded import it in Unity either with a double click on the file or from Unity `Assets > Import Package > Custom Package...`. Select all and click `Import`.
+
+**NOTE:** If you are working with an existing Cardboard project delete the *GoogleVR* and *Plugins* folders from the *Assets* folder before importing.
+
+### 6. Test with Google Demo App
+Now let's see if everything is working fine!
+Open the *ControllerDemo* scene (`GoogleVR > DemoScenes > ControllerDemo > ControllerDemo`). You should see this:
+
+<img
+  class="modal-image"
+  src="/img/blog/0009/unity_controller_demo_scene.PNG"
+  alt="article-img"
+  title="Unity Player Settings">
+
+I didn't heard of any way to have the controller working in Unity emulator (unless you use a phone to [emulate the controller](https://developers.google.com/vr/daydream/dev-kit-setup "Emulate the Daydream controller"), but the point here is to use the actual controller...). So you will have to build the app to your Pixel.
+
+Remember to change the `Bundle Identifier` in the *Player Settings*, like for any other Android app in Unity. Then click `File > Build & Run`.
+
+**NOTE:** Like for any Android app you need to *Enable USB debugging on your device* as described [here](https://docs.unity3d.com/Manual/android-sdksetup.html "Enable USB debugging on your device").
+
+When it's done building, follow the instructions on your phone. The app features several cubes that turn orange when you move the pointer over them. You can move them around by touching and holding the touchpad when pointing at one.
+
+And here we are! You are all set up to create your how apps!
+
+
+## Gimme more!
+Here are a few other stuffs you could find interesting.
+
+### Good to read
+1. [Google's documentation](https://developers.google.com/vr/unity/controller-basics "Google's Daydream controller documentation") for Daydream controller.
+
+2. Want to publish an app on Daydream special store? You will need to register to [Daydream Access Program](https://developers.google.com/vr/daydream/daydream-access-program "Daydream Access Program").
+
+### Test our quick Daydream game prototype!
+In the 2 days that followed our reception of the controller I put together a small game for Daydream.
+You can get it for free from Google Play [here](https://play.google.com/store/apps/details?id=com.tengio.the_chickens_strike_back "The Chickens Strike Back").
+We plan on adding a lot more content and better art (whenever we get the time ^^'). Here it's mainly done with free models.
+
+### Remove part of the useless controller emulator stuffs
```

Comment:
Remove part of the useless controller emulator stuffs =>  Remove part of the useless controller emulator BITS